tetrateci/version_check.py

```diff
@@ -3,9 +3,10 @@
 import sys, os
 
 version_matrix = {
-    "1.7" : {"1.16", "1.17", "1.18"},
-    "1.8" : {"1.16", "1.17", "1.18", "1.19"},
-    "1.9" : {"1.17", "1.18", "1.19", "1.20"},
+    "1.7": {"1.16", "1.17", "1.18"},
+    "1.8": {"1.16", "1.17", "1.18", "1.19"},
+    "1.9": {"1.17", "1.18", "1.19", "1.20"},
```

Comment:
Does it not work on k8s 1.16?

Reply:
Not supported according to [this](https://istio.io/v1.9/docs/setup/platform-setup/)